float32 search_radius
---
bool success